cache.js

```diff
@@ -42,7 +41,7 @@ function saveFileToPRCache (repoID, prID, filePath, data, fileType) {
  * @param {number} prID pull request identifier
  */
 function clearPRCache (repoID, prID) {
-  fs.removeSync(getBranchPath(repoID, prID, 'bandit'))
+  fs.removeSync(getBranchPath(repoID, prID, 'other linters'))
```

Comment:
Elsewhere in the code we say `'other linter'` not `'other linters'`, might be worth defining a constant?

Reply:
That's not a bad idea.
It's connected with this issue: 
https://github.com/vmware/precaution/issues/103

This is an optional parameter. Maybe I can just remove it?
I place it there to be more explicit. What do you think?

Reply:
Personally I'd remove it, the function documentation lists the parameter as optional.
Btw the docstring needs updating, it says:
`@param {string} app Optional: an app that calls the function. (default bandit)`
but the default has now changed

Reply:
Maybe a better way to do this is by making this argument as a flag.
We need that argument because of how Golang handles paths and that it has to use GOPATH.

Maybe a good idea will be to create this as a boolean flag.
If it's up it can mean "that's a golan files" or "gosec".

Reply:
I didn't mean to suggest we remove the parameter from the function entirely. Rather instead we could explicitly set the default parameter using JavaScript's [default function parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/default_parameters) support and then only pass the parameter when we need the non-default.

A boolean flag wouldn't be the best idea, because we'd then be exposing backend specific functionality in a generic function.